src/.nuget/packageLoad.targets

```diff
@@ -38,31 +38,17 @@
   <Target Name="_RestoreBuildToolsPackagesConfig"
```

Comment:
Should this target have a dual xplat / master behaviour? On xplat it uses CLI and project.json, on master it uses packages.config. Or should we just pick the CLI from buildtools for both?